Vostok.Metrics.System/Host/NativeHostMetricsCollector_Linux.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using Vostok.Metrics.System.Helpers;
+
+// ReSharper disable PossibleInvalidOperationException
+
+namespace Vostok.Metrics.System.Host
+{
+    internal class NativeHostMetricsCollector_Linux : IDisposable
+    {
+        private readonly ReusableFileReader systemStatReader = new ReusableFileReader("/proc/stat");
+        private readonly ReusableFileReader memoryReader = new ReusableFileReader("/proc/meminfo");
+        private readonly HostCpuUtilizationCollector cpuCollector = new HostCpuUtilizationCollector();
+
+        public void Dispose()
+        {
+            systemStatReader.Dispose();
+            memoryReader.Dispose();
+        }
+
+        //https://supportcenter.checkpoint.com/supportcenter/portal?eventSubmit_doGoviewsolutiondetails=&solutionid=sk65143
+        public void Collect(HostMetrics metrics)
+        {
+            var systemStat = ReadSystemStat();
+            var memInfo = ReadMemoryInfo();
+
+            if (systemStat.Filled)
+            {
+                var systemTime = systemStat.UserTime.Value + systemStat.NicedTime.Value + systemStat.SystemTime.Value +
+                                 systemStat.IOWaitTime.Value + systemStat.InterruptsTime.Value +
+                                 systemStat.SoftInterruptsTime.Value;
+
+                cpuCollector.Collect(metrics, systemTime, systemStat.IdleTime.Value);
```

Comment:
Something's amiss with this calculation. Here's a comparison of process vs host metrics in a Docker container with 2 cores and a process that consumes 1 core:
```
[CurrentProcessMetrics] CPU = 50.75% (1.02 cores) ...

Host metrics:
CpuUtilizedCores: 0.06862745098039214
CpuUtilizedFraction: 0.03431372549019607
```